### DISCUSSION TITLE
LinkLiar 3.0.3 have been released

### DISCUSSION THREAD

**Discussion by Supremesir** [Homebrew/discussions#2267] [Casks] [answered]
[LinkLiar](https://github.com/halo/LinkLiar/releases) have a new stable version 3.0.3. 
But his daemon executable directory changed, I don't know how this will affect homebrew-cask. So I can't simply change the [linkliar.rb](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/linkliar.rb) file and pr. I hope some experienced people can verify and submit. Thanks.

**Comment by SMillerDev** [comment]
You can make the pull request and mention this. Then maintainers can help if it's needed.

**Comment by neutric** [comment, marked as answer]
@Supremesir See https://github.com/Homebrew/homebrew-cask/pull/113857. Please mark this as answered.
